docs/Wiki/ProductAndTestProjects.md

```diff
@@ -1,29 +1,60 @@
-﻿This page refers to the SonarScanner for .NET, also known as SonarScanner for MSBuild (S4MSB). You can read more about how to install and use it in the ["SonarScanner for .NET documentation"](https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html).
+﻿This page refers to the SonarScanner for .NET (S4NET), also known as SonarScanner for MSBuild (S4MSB). You can read more about how to install and use it in the ["SonarScanner for .NET documentation"](https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html).
```

Comment:
S4NET seems a bit weird, we're missing the DOT (.) somehow.

Reply:
It's not perfect, but it's the best of available options I can see.
`S4MSB` - doesn't reflect current name anymore
`S4.NET` - I don't see the Scanner part here, looks more like URL or some .NET library
`S4DOTNET` - looses the advantage of a short abbreviation
`S4DNET` - doesn't remind the dot

Is there some better option?

Reply:
My first thought was maybe S4.NET or S4DN but yep, this sounds a bit weird. We can even call it "SANETTT" with that, funny !